Cargo.lock

```diff
@@ -16613,3 +16530,8 @@ dependencies = [
  "libc",
  "pkg-config",
 ]
+
+[[patch.unused]]
+name = "quanta"
+version = "0.11.0"
+source = "git+https://github.com/madsim-rs/quanta.git?rev=948bdc3#948bdc3d4cd3fcfe3d52d03dd83deee96d97d770"
```

Comment:
I guess this might break madsim

Reply:
Is there a way to fix it? Directly removing the part does not help.

Reply:
after removing 
```toml
# from Cargo.toml, in `[patch.crates-io]`
quanta = { git = "https://github.com/madsim-rs/quanta.git", rev = "948bdc3" }
```
the `[[patch.unused]]` section disappears. But the tests still stuck

```
  Running [ 00:01:43] [=============================================================================>                                                                                            ]  533/1167: 12 running, 533 passed (4 slow), 22 skipped     
   Canceling due to interrupt: 12 tests still running
   Canceling [ 00:01:43] [=============================================================================>                                                                                            ]  533/1167: 12 running, 533 passed (4 slow), 22 skipped     
      SIGINT [ 103.327s] risingwave_batch executor::generic_exchange::tests::test_exchange_multiple_sources
      SIGINT [ 102.891s] risingwave_batch executor::merge_sort_exchange::tests::test_exchange_multiple_sources
      SIGINT [ 102.802s] risingwave_batch executor::tests::test_clone_for_plan
      SIGINT [ 102.726s] risingwave_batch task::task_manager::tests::test_task_id_conflict
      SIGINT [  83.848s] risingwave_frontend binder::bind_param::test::subquery
      SIGINT [  83.893s] risingwave_frontend binder::bind_param::test::cast_after_specific
      SIGINT [  83.899s] risingwave_frontend binder::bind_param::test::basic_value
      SIGINT [  83.887s] risingwave_frontend binder::bind_param::test::default_type
      SIGINT [  83.903s] risingwave_frontend binder::bind_param::test::basic_select
      SIGINT [  83.042s] risingwave_frontend binder::expr::value::tests::test_bind_radix
      SIGINT [  83.886s] risingwave_frontend binder::bind_param::test::infer_case
      SIGINT [  83.498s] risingwave_frontend binder::expr::value::tests::test_bind_interval
------------
```

the dep to `quanta` is introduced by `moka`

Reply:
Patching `quanta` is **required**, not unexpected.

moka 0.12.0 depends on quanta 0.11, but you upgraded moka to 0.12.8, which depends on quanta 0.12. Therefore, original quanta is used instead of the patched one. That's why it breaks.

<img width="762" alt="image" src="https://github.com/user-attachments/assets/608cbd62-8afc-452d-a3cf-48538ad87a5b">

Reply:
You should either revert to moka 0.12.0, or upgrade the madsim quanta to 0.12